src/core/src/package_managers/AptitudePackageManager.py

```diff
@@ -47,6 +48,13 @@ def __init__(self, env_layer, composite_logger, telemetry_writer, status_handler
         # Package manager exit code(s)
         self.apt_exitcode_ok = 0
 
+        # auto OS updates
+        self.update_package_list = 'APT::Periodic::Update-Package-Lists'
+        self.unattended_upgrade = 'APT::Periodic::Unattended-Upgrade'
+        self.image_default_patch_mode_file_path = '/etc/apt/apt.conf.d/20auto-upgrades'
```

Comment:
The term image_default is overloaded here and causes confusion when reading code and function names further below.

image_default = configuration the image started off with. The file you're looking at is just os_patch_mode_settings_file_path (emphasis on os and settings for clarity). That file simply happen to have what we consider the image_default settings in the beginning.